public/js/calendar.js

```diff
@@ -92,10 +113,34 @@ Cal.prototype.showMonth = function (y, m) {
     var chk = new Date();
     var chkY = chk.getFullYear();
     var chkM = chk.getMonth();
-    if (chkY === this.currYear && chkM === this.currMonth && i === this.currDay) {
+    if (
+      chkY === this.currYear &&
+      chkM === this.currMonth &&
+      i === this.currDay
+    ) {
       html += '<td data-date=' + dstring + ' class="day today">' + i + '</td>';
     } else {
-      html += '<td data-date=' + dstring + ' class="day normal">' + i + '</td>';
+      var currentDate = new Date(this.currYear, this.currMonth, i);
+      for (var h = 0; h < dateArray.length; h++) {
```

Comment:
I know this is someone else's code but can we clean it up a little..? Make this fella a forEach and clear up some of the variable names :pray:

Reply:
cause im using a ` break; ` when i actually find a match with the date